.github/workflows/build-docs.yml

```diff
@@ -41,20 +44,24 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
 
-      - name: Install Doxygen (linux only)
+      - name: Install Doxygen
         run: |
           set -e pipefail
           sudo apt-get update
           # Install doxygen *before* running cmake
           sudo apt-get install -y  doxygen
           pip install virtualenv
         shell: bash
-        if: ${{ runner.os == 'Linux' }}
 
-      - name: Build Doxygen Docs (linux only)'
+      - name: Build Doxygen Docs
         run: |
           # Build the documentation (this does not deploy to RTD).
           cd $GITHUB_WORKSPACE/tiledb/doxygen;
           ./local-build.sh;
         shell: bash
-        if: ${{ runner.os == 'Linux' }}
+
+      - name: Upload Doxygen Docs
```

Comment:
This step changes the workflow. We should not combine changes like this with simple code rearrangements.

Part of the reason to segregate substantive changes from rearrangements is to avoid delay. This step doesn't even seem like a good idea. Rather than debate it here and delay the work, drop it from this PR.